dotcom-rendering/src/web/server/pageTemplate.ts

```diff
@@ -61,14 +61,13 @@ export const pageTemplate = ({
 		dataObject: { [key: string]: string },
 		attributeName: 'name' | 'property',
 	) => {
-		if (dataObject) {
-			return Object.entries(dataObject)
-				.map(
-					([id, value]) =>
-						`<meta ${attributeName}="${id}" content="${value}"/>`,
-				)
-				.join('\n');
-		}
+		return Object.entries(dataObject)
+			.map(
+				([id, value]) =>
+					`<meta ${attributeName}="${id}" content="${value}"/>`,
+			)
+			.join('\n');
+
 		return '';
```

Comment:
I think this return is not necessary, and you can even drop the curly braces altogether!